### ISSUE TITLE
User defined simulation tstart

### ISSUE THREAD

**Issue by ntolley** [jonescompneurolab/hnn-core#379]
This was brought up by @jasmainak during #348, it would be nice to be able to define a negative tstart for plotting. This is especially relevant for event related potentials, where the peaks are named by clinical electrophysiologists according to when they appear in time relative to a stimulus.

At a first pass, it seems like one of the easiest workarounds would be to have a `tstart` attribute of the network, and substract the from the `.times`attribute of the various objects that store it during plotting. I believe there is some internal tests/logic built into `CellResponse.times` for example that may break if we just edit the `.times` attribute directly.

**Comment by jasmainak** [comment]
This still looks pretty relevant if someone wants to give it a shot

**Comment by ntolley** [comment]
I guess where this gets weird is that the length of the simulation will change depending on `tstart`. Are you thinking of just `simulate_dipole(net, ..., tstart=-50.0)` or something?

**Comment by jasmainak** [comment]
yep, that's exactly the API I was imagining ... one would have to be carefully adjust the times when adding the drives as well, not just the cell responses. E.g.,

```py
net.add_evoked_drive(mu=20.)  # 20 ms or whatever is the conduction delay 
```

for an N20 response ...
